requirements.txt

```diff
@@ -34,3 +34,4 @@ sqlparse==0.4.4
 typing_extensions==4.8.0
 urllib3==2.0.7
 whitenoise==6.6.0
+responses==0.24.1
```

Comment:
Perfect! Thank you for adding this to the requirements.txt!

Reply:
np!

Reply:
Awesome job throughout!